aws-crt-kotlin/build.gradle.kts

```diff
@@ -116,6 +121,37 @@ kotlin {
     sourceSets.all {
         optinAnnotations.forEach { languageSettings.optIn(it) }
     }
+
+    // create a single "umbrella" cinterop will all the aws-c-* API's we want to consume
+    // see: https://github.com/JetBrains/kotlin-native/issues/2423#issuecomment-466300153
+    targets.withType<KotlinNativeTarget> {
```

Comment:
**Question**: I don't quite understand the referenced comment or how it applies to this interop config. Why is a single "umbrella" cinterop better than individual cinterops per library?